ykrt/src/deopt.rs

```diff
@@ -160,9 +160,8 @@ struct ReconstructInfo<'a> {
     nfi: Option<NewFramesInfo>,
 }
 
-// Function containing the actual code setting up the frame reconstructor. This function will be
-// called by `LLVMOrcThreadSafeModuleWithModuleDo` passing relevant info using the
-// `ReconstructInfo` struct.
+/// Actual code for reconstructing the stack. Called from `__ykrt_deopt` via
```

Comment:
"Actual code" feels redundant. Can we make the first sentence more action orientated? So something like "Reconstruct the stack." I think the "Called from" bit makes clear enough that there are intermediary functions.

Reply:
@ptersilie Ping.

Reply:
Try this: [6329281](https://github.com/ykjit/yk/pull/780/commits/632928150a87d44d7f0c8d106e10b4d50a416d63)